### ISSUE TITLE
getAspectRatio returns incorrect / different value.

### ISSUE THREAD

**Issue by mijewe** [mikestecker/craft3-videoembedder#26]
Previously, I used getAspectRatio to return the padding-top I'd need to use to make my embed scale responsively.

So for a 16:9 video, that would be 56 (%).
That's calc'd by height/width*100.

Now (I believe since 1.1.4), this returns 1.64.
That's calc'd by width/height.

Though I suppose technically this is the aspect ratio, it's not useful in same way it was before.

So that's question 1: Should the value be w/h, or h/w?

Question 2:
I tried to do maths in the template to resolve the issue (1 / aspectRatio * 100), but because the getAspectRatio function returns a Twig_Markup object, I wasn't able to convert it to a float in order to do the maths.

My temporary solution was to add a custom variable to a different module.
```
return 1 / VideoEmbedder::$plugin->service->getInfo($url)->aspectRatio * 100;
```

**Comment by mijewe** [comment]
On further investigation:

YouTube videos are returning a percentage ratio (height / width * 100).
Vimeo URLs are returning the new decimal ratio (width / height).

**Comment by mijewe** [comment]
Seems like other functions are also broken, like `getProviderName()`. Inconsitent results between Vimeo and YouTube.

For now, my temp fix is:
```
    public function getAspectRatio($url) {

        $isVimeo = VideoEmbedder::$plugin->service->isVimeo($url);

        if ($isVimeo) {
            return 1 / VideoEmbedder::$plugin->service->getInfo($url)->aspectRatio * 100;
        } else {
            return VideoEmbedder::$plugin->service->getInfo($url)->aspectRatio;
        }

    }
```

**Comment by mikestecker** [comment]
I haven't checked anything in quite a while with this plugin so feel free to send me some merge requests if you know how to fix any issues.

I did think this one was strange though that it would return different results like that. The plugin is using another composer library to pull this data which should always return a whole number if I'm reading their function correctly: https://github.com/oscarotero/Embed/blob/c8e1514861e7880f31e84b2c883bf31cff3e182a/src/Adapters/Adapter.php#L567

I could be wrong obviously, my PHP understanding is very low still and I have been out of working with Craft for quite some time but I'm slowly getting back up to speed. I'll keep looking into this and other potential bugs.

**Comment by mijewe** [comment]
Yeah I will do - just need to find the time in between work work.

You're probably right about that Embed library, but then in https://github.com/mikestecker/craft-videoembedder/commit/c5078ace38f92ecadb119c7d25155d73c0e389bc#diff-6881687061cd7659222de9298625ce53 it got overridden.

**Comment by engram-design** [comment]
Sorry to say that while we've taken over this plugin with the intention to bring it to Craft 4+, we're not going to be continuing development on this plugin. In addition, we're no longer supporting Craft 3 plugins due to Craft itself reaching its EOL.

But the good news is we've got an even better plugin (we think) for Craft 4 and Craft 5 called [Video Picker](https://verbb.io/blog/video-picker) which has the same [embeddable videos](https://verbb.io/craft-plugins/video-picker/docs/template-guides/embedding-videos) feature.

The even better news is that we're happy to gift anyone a **free** license for any site they have with Video Embedder installed, to ease the migration to Craft 4/5. Just [get in touch](https://verbb.io/contact) with a screenshot of your current site with Video Embedder installed as proof of upgrade.

This repository will still exist as a reference, PR's can still be made, and releases completed, but there won't be any active development from us.
